### PR TITLE
Changed the caption description

### LINKED ISSUE #3: Activity 1: Your First Caption
Using the API, I have just created a file with a random octocat for you to caption. In this activity, we will learn how to make changes to files in our GitHub repository.

- [ ] @c4llmeco4ch
- [ ] @DekalbIrina
- [ ] @InHisGrace76
- [ ] @NNSU99
- [ ] @ajo-github
- [ ] @lumapathy
- [ ] @mcajuste01
- [ ] @samkrishnan
- [ ] @sammyseltzer

### PR DESCRIPTION
**Pull request by ajo-github**
# Editing Things

- Removed old caption

### CODE REVIEW

File: _slides/06-ajo-github.md [hunk 1]
```diff
@@ -3,7 +3,7 @@ layout: slide
 title: "Welcome to our slide deck!"
 ---
 
-Caption here
+DOTA 2 is an difficult but fun MOBA.
```
Comment (InHisGrace76) [inline _slides/06-ajo-github.md:6]: Good Job!

**Review by InHisGrace76 — COMMENTED** [review]
On to the Next Phase!

**Comment by InHisGrace76** [comment]
Looks good to me!

File: _slides/06-ajo-github.md (hunk 1 above)
Comment (samkrishnan) [inline _slides/06-ajo-github.md:6]: This is a brilliant line.. This line should be in a movie...

**Review by samkrishnan — APPROVED** [review]
This is where the real magic happens - when we approve it..

**Pull request closed without merging**